examples/ReactJsSdkApp/README.md

```diff
@@ -1,17 +1,25 @@
-Install all the dependencies of this REACT application with `yarn install` 
+# REACT example application
 
-Start the client with `yarn start`
+## Frontend
+Use the project `ReactJsSdkApp`:
 
-Then you need a backend to generate the tokens.
+Install all the dependencies of this REACT application. `yarn install`\
+Start the client. `yarn start`\
+Run the backend to generate the tokens.
 
-Use the project `PythonBackendApp`, start it like that :
+## Backend
+Use the project `PythonBackendApp`:
 
+Start the backend.
 ```
 python3 client_backend.py -sk <path to private key> -pk <PTI public key> -c <client id i.e. be623155-18f7-40fc-99ed-27660c14e513>
 ```
 
 Then you can trigger the popup closure with a curl request on the backend, like this :
-
 ```
 curl -X POST -d '{"requestId":"<id of the request>"}' http://localhost:5000/webhook
 ```
+
+> ***N.B.*** When developing, we all use the same existing client account: "cucumber". \
+Ask the other devs, if you don't have the private & public key for this account. \
+To know the clientId of the "cucumber" client, go on the `pti-dev-clients.client` database and take note of its id.
```

Comment:
clientId is always the same : 3f716642-1955-11eb-adc1-0242ac120002

Reply:
Je me demandais si c'était une information qui pouvait être mise directe sur github. :)

Reply:
le clientId est déjà publié dans la sample react app.